### PR TITLE
Add document about stepwise tuning for LightGBM.

### PR DESCRIPTION
**Pull request by toshihikoyanase**
This PR adds a document of `optuna.integration.lightgbm.train`.

image

### CODE REVIEW

File: optuna/integration/lightgbm_tuner/__init__.py [hunk 1]
```diff
@@ -11,9 +11,15 @@
 
 def train(*args, **kwargs):
     # type: (List[Any], Optional[Dict[Any, Any]]) -> Any
-    """Wrapper function of LightGBM API: train()
+    """Wrapper of LightGBM Training API to tune hyperparameters.
 
-    Arguments and keyword arguments for `lightgbm.train()` can be passed.
+    .. warning::
+
+        This feature is experimental. The interface can change in the future.
```
Comment (c-bata) [inline optuna/integration/lightgbm_tuner/__init__.py]: It looks passive voice is appropriate.

```suggestion
        This feature is experimental. The interface can be changed in the future.
```

Comment (c-bata) [reply optuna/integration/lightgbm_tuner/__init__.py]: [nits] "may be changed" might be more suitable.

Comment (toshihikoyanase) [reply optuna/integration/lightgbm_tuner/__init__.py]: Thank you! I accepted your suggestion.

[TODO] I'll also fix the following comment:
https://github.com/pfnet/optuna/blob/b96a3960ba8a6b636c18a7999e9ab819e6051c83/optuna/integration/sklearn.py#L337
At that time, I'll replace `can` with `may`.

**Comment by toshihikoyanase** [comment]
I updated the message based on [this discussion](https://github.com/pfnet/optuna/pull/635#discussion_r338402199). The same change will also be applied to `OptunaSearchCV` in https://github.com/pfnet/optuna/pull/639.

**Pull request merged**